run.py

```diff
@@ -74,16 +76,21 @@ def call_compiler(scalaLocation, output, additionalOptions=[]):
                     scalacOptions +
                     sources +
                     debugOptions +
+                    additionalScalacOptions +
                     options.additionalOptions)
+    return time.time() - timeBefore
 
 
-call_compiler(options.scala, scalaOutput)
+compilation_time = call_compiler(options.scala, scalaOutput, ["-Yparallel-phases:parser", "-Yparallel-threads", "16"])
```

Comment:
Revert. We want to be able to test this with different phases, different amount of threads, sometimes without threads etc